client/components/navigation-header/style.scss

```
@@ -4,7 +4,6 @@
 .navigation-header {
 	box-sizing: border-box;
 	width: 100%;
-	background-color: var(--studio-white);
 	padding: 16px 16px 0 16px;
```

Suspect we need: 

```suggestion
	margin: 16px;
```

And to remove https://github.com/Automattic/wp-calypso/pull/83066/files?w=1#diff-04938ea65060f21a870ab4cae23fe73e66dfbc1362b14b3deebab9eb553f0b0bR58